src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -2296,6 +2300,15 @@ private void StartCommandLoopOnRunspaceAvailable()
 
         private void OnDebuggerStop(object sender, DebuggerStopEventArgs e)
         {
+            // We maintain the current stop event args so that we can use it in the DebugServer to fire the "stopped" event
+            // when the DebugServer is fully started.
+            CurrentDebuggerStopEventArgs = e;
+
+            if (!IsDebugServerActive)
+            {
+                _languageServer.SendNotification("powerShell/startDebugger");
```

Comment:
Is there a constant we could use here?

Reply:
Yeah Omnisharp has a type called "EventMethods" that contain all the constant names of the messages.

Ill move over to that in the next PR.